src/functionaltests/java/com/ericsson/ei/utils/FunctionalTestBase.java

```diff
@@ -182,40 +175,36 @@ protected JsonNode getJSONFromFile(String filePath) throws IOException {
 
     /**
      * Verify that aggregated object contains the expected information.
-     * 
-     * @param arguments
-     *            list of arguments to check
-     * @return list of missing arguments
+     *
+     * @param checklist list of checklist to check
+     * @return list of missing checklist
      * @throws InterruptedException
      */
-    protected List<String> verifyAggregatedObjectInDB(List<String> arguments) throws InterruptedException {
-        List<String> checklist = new ArrayList<String>(arguments);
-        MongoClient mongoClient = new MongoClient(getMongoDbHost(), getMongoDbPort());
+    protected List<String> verifyAggregatedObjectInDB(List<String> checklist) throws InterruptedException {
```

Comment:
Since you renamed "checklist" variable in other places,, use same variable name here also.

Reply:
i just removed unneeded list